File: psup assignment/cpp/q20.cpp
#include <iostream>
#include <string>
#include <cctype>
int main() {
    int choice;
    std::cout << "\n--- Program 20: Beverage Menu ---\n";
    std::cout << "Enter beverage choice (1: Tea, 2: Coffee, 3: Juice): ";
    std::cin >> choice;
    switch (choice) {
        case 1: std::cout << "Tea" << std::endl; break;
        case 2: std::cout << "Coffee" << std::endl; break;
        case 3: std::cout << "Juice" << std::endl; break;
        default: std::cout << "Invalid choice" << std::endl; break;
    }
    return 0;
}

File: psup assignment/java/q20.java
import java.util.*;
class BeverageMenu {
    public static void main(String[] args) {
        Scanner input = new Scanner(System.in);
        System.out.print("Enter beverage choice (1: Tea, 2: Coffee, 3: Juice): ");
        int choice = input.nextInt();
        switch (choice) {
            case 1: System.out.println("Tea"); break;
            case 2: System.out.println("Coffee"); break;
            case 3: System.out.println("Juice"); break;
            default: System.out.println("Invalid choice"); break;
        }
        input.close();
    }
}